### DISCUSSION TITLE
How to change the directory gauge looks for the "env" folder?

### DISCUSSION THREAD

**Discussion by guntribam** [getgauge/gauge#1972] [General]
If "env" folder is not on my project root gauge ends up using the global default properties instead of my local properties. How can i change "env" folder path inside my project and tell gauge the new path?

**Comment by zabil** [comment]
>  tell gauge the new path

Not very clear on what you mean by this. But there's documentation about it here https://docs.gauge.org/configuration.html?os=linux&language=javascript&ide=vscode#using-environments-in-a-gauge-project

**Reply by guntribam** [reply]
Sorry for not being clean. This is the env folder(on a gauge-js project):

env
|--default.properties
|--js.properties

If a i execute ``` gauge run specs ``` on root level, gauge will search for the "env" folder on the root level of my project.

The thing is that i want to organize my tests like this:

tests
|--env 
|--docs
|--reports
|--logs
|--specs
|--step_implementation.js

I want to put everything on a folder called tests that will be on the root level of my project, but when i execute gauge with this layout it uses the properties defined on my global .gauge folder and cannot find ``` ./tests/env ``` on my project. How can i tell gauge to look for the env folder inside tests folder?

**Reply by sriv** [reply]
So, if I understand right, your gauge tests are in a subfolder and you would like to invoke gauge from the root folder?

If so, you can set the working directory for gauge using the `--dir` flag. ex: `gauge run specs --dir=tests`. Note that all the folder convention would be followed, ex: logs and reports will be created relative to `tests` in this case.

**Reply by guntribam** [reply]
Thanks a lot.
